internal/config/config.go

```diff
@@ -65,6 +65,10 @@ type HttpConfig struct {
 	IDLETimeout                  time.Duration `env:"HTTP_IDLE_TIMEOUT,default=10s"`
 	DisableKeepAlive             bool          `env:"HTTP_DISABLE_KEEP_ALIVE,default=true"`
 	DisablePreParseMultipartForm bool          `env:"HTTP_DISABLE_PARSE_MULTIPART_FORM,default=true"`
+
+	SSOEnabled      bool     `env:"SSO_ENABLED,default=false"`
```

Comment:
Should we rename this settings to `SSOProxyAuth...`? In case we ever support more than one SSO auth method in the future.